charts/sda-svc/templates/auth-deploy.yaml

```
@@ -196,6 +200,13 @@ spec:
                   - key: {{ required "The name of the JWT signing key is needed" .Values.global.auth.jwtKey }}
                     path: {{ .Values.global.auth.jwtKey }}
       {{- end }}
+      - name: c4gh
+        - secret:
+          defaultMode: 0440
+          secretName: {{ required "A secret for the C4GH public key is needed" .Values.global.c4gh.secretName }}
+          items:
+            - key: {{ required "The C4GH public key is needed" .Values.global.c4gh.publicFile }}
+              path: {{ .Values.global.c4gh.publicFile }}
```

```suggestion
      {{- if not .Values.global.vaultSecrets }}
        - name: c4gh
          - secret:
            defaultMode: 0440
            secretName: {{ required "A secret for the C4GH public key is needed" .Values.global.c4gh.secretName }}
            items:
              - key: {{ required "The C4GH public key is needed" .Values.global.c4gh.publicFile }}
                path: {{ .Values.global.c4gh.publicFile }}
      {{- end }}
```

In correct indentation was the root cause of the issue